### ISSUE TITLE
[Core][Nagrand] Halaa Battle Tokens

### ISSUE THREAD

**Issue by KataRinne** [dalaranwow/dalaran-wow#2272]
I tested the system with [Halaa Battle Tokens](http://www.wowhead.com/item=26045/halaa-battle-token) in this server, and I found out that, currently, these items are unobtainable. Because of that, players are unable to obtain some of the best-looking ground mounts: [Dark Riding Talbuk](http://www.wowhead.com/item=28915/reins-of-the-dark-riding-talbuk) & [Dark War Talbuk](http://www.wowhead.com/item=29228/reins-of-the-dark-war-talbuk).

**How to fix:**
Basically it should be possible to get a Halaa Battle Token by killing a player from an opposite faction in Halaa, which is in Nagrand (If his level is not gray to you).

**Comment by ImprezaWoW** [comment]
Obtainable, might have been fixed along the way or was never an issue.
